packages/core/src/errors/errors.ts

```diff
@@ -4,9 +4,19 @@ export enum MomentoErrorCode {
   // Service returned an unknown response
   UNKNOWN_SERVICE_ERROR = 'UNKNOWN_SERVICE_ERROR',
   // Cache with specified name already exists
+  CACHE_ALREADY_EXISTS_ERROR = 'ALREADY_EXISTS_ERROR',
+  /** @deprecated use CACHE_ALREADY_EXISTS_ERROR instead */
   ALREADY_EXISTS_ERROR = 'ALREADY_EXISTS_ERROR',
+  // Store with specified name already exists
+  STORE_ALREADY_EXISTS_ERROR = 'ALREADY_EXISTS_ERROR',
   // Cache with specified name doesn't exist
+  CACHE_NOT_FOUND_ERROR = 'NOT_FOUND_ERROR',
+  /** @deprecated use CACHE_NOT_FOUND_ERROR instead */
   NOT_FOUND_ERROR = 'NOT_FOUND_ERROR',
+  // Store with specified name doesn't exist
+  STORE_NOT_FOUND_ERROR = 'STORE_NOT_FOUND_ERROR',
+  // Item with specified key doesn't exist
+  ITEM_NOT_FOUND_ERROR = 'ITEM_NOT_FOUND_ERROR',
```

Comment:
We should call this `STORE_ITEM_NOT_FOUND_ERROR` to make clear this applies to the storage service and not cache.